dashboard/src/actions/endpointAction.js

```diff
@@ -0,0 +1,25 @@
+import * as TYPES from "./types";
+
+export const fetchEndpoints = () => async (dispatch) => {
+  try {
+    const response = await fetch("/api/v1/endpoints");
+
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${response.status}`);
+    }
+    const data = await response.json();
+    window.endpoints = data;
+    dispatch({
+      type: TYPES.SET_ENDPOINTS,
+      payload: { ...data },
+    });
```

Comment:
So we're saving the endpoints object both at `window.endpoints` and at `apiEndpoings` in Redux. Since your components are using Redux to access the configuration, I don't think we need the `window.endpoints` copy.

Reply:
Removed the `window.endpoints` copy